### PR TITLE
configure restarter hook for aldryn-platform

### PR DESCRIPTION
**Pull request by stefanfoulis**
enables a webhook if the cms is in need of a restart due to changes in
apphook configuration.

### CODE REVIEW

**Comment by stefanfoulis** [comment]
same as https://github.com/aldryn/aldryn-django-cms/pull/12 , but for the `support/3.2.x` branch. I added a comment that this should be completely removed in 3.2.x, once we've tested/fixed the builtin reload functionality.

**Comment by cyberstar** [comment]
LGTM

File: aldryn_django_cms/models.py [hunk 1]
```diff
@@ -3,18 +3,29 @@
...
+RESTARTER_URL = getattr(settings, 'RESTARTER_URL', None)
+RESTARTER_PAYLOAD = getattr(settings, 'RESTARTER_PAYLOAD', None)
+
+
 def trigger_server_restart(**kwargs):
     # internal endpoint to trigger safe restart
-    restarter_url = getattr(settings, 'RESTARTER_URL', None)
+    restarter_url = RESTARTER_URL
 
     if restarter_url:
```
Comment (cyberstar) [inline aldryn_django_cms/models.py:21]: why not use `if RESTARTER_URL:` ? 
though this is very minor improvement...

Comment (stefanfoulis) [inline aldryn_django_cms/models.py:21]: right... after no longer using `getattr` here, saving to `restarter_url` is pointless.

**Pull request merged**